<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForm</class>
 <widget class="QWidget" name="OrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Распоряжения на получение (отправку) спецтехники</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Журнал учета распоряжений</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget_journal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>130</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Номер</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Дата</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Номер</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Добавить новое распоряжение</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="lineEdit_number"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Дата</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_add_order">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDateEdit" name="dateEdit">
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Содержание распоряжения</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget_order">
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>№№ п/п</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Отправитель</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Получатель</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата отправки</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Наименование груза</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Кол-во</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>СГ</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Блокировка</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Вид транспорта</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_transport"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Ответственный за перевозку:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_responsib"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Особые указания</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_spec_orders">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_content">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_del">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_print">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exit">
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_number</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_add_order</tabstop>
  <tabstop>comboBox_transport</tabstop>
  <tabstop>comboBox_responsib</tabstop>
  <tabstop>textEdit_spec_orders</tabstop>
  <tabstop>pushButton_content</tabstop>
  <tabstop>pushButton_del</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_exit</tabstop>
  <tabstop>tableWidget_order</tabstop>
  <tabstop>tableWidget_journal</tabstop>
 </tabstops>
 <resources>
  <include location="../images/transport_control.qrc"/>
 </resources>
 <connections/>
</ui>
